types/host.go

```diff
@@ -98,7 +98,7 @@ type OSInfo struct {
 // LoadAverage is the interface that wraps the LoadAverage method.
 // LoadAverage returns load info on the host
 type LoadAverage interface {
-	LoadAverage() LoadAverageInfo
+	LoadAverage() (*LoadAverageInfo, error)
```

Comment:
Please add a changelog entry in CHANGELOG.md and note that is a breaking API change.

Reply:
Technically this is new APi that didn't exist before that _should_ have. So this is not a breaking APi change IMO, it _should_ be backwards compatible. Yes the "interface" has changed, but that will only affect new projects? Even old projects would still compile AFAIK.

Reply:
It is a breaking change. The interface already existed. Any users that had written code to use it will be broken by this change. It does not matter that there was no implementation, code that was written to expect `LoadAverage() LoadAverageInfo` won't compile now.

But given that there was no actual implementation I doubt that any code exists that actually tried to use it. And because of this, I'm OK with make this change to the interface. It would be a good idea to do a search on cs.github.com to see if we can find any usages.

Reply:
Yeah that was kind of my point, but you put it better 👌

Reply:
Added to the `CHANGELOG.md`.

Please, have a look, maybe the explanation is a bit odd since I'm not a native speaker.